src/_nebari/deploy.py

```diff
@@ -73,3 +73,5 @@ def deploy_configuration(
         print(
             "Additional administration docs can be found at https://docs.nebari.dev/en/stable/source/admin_guide/"
         )
+
+    return stage_outputs
```

Comment:
Why do we need to return stage_outputs now?

Reply:
For testing, it makes it easier to test the state of the deployment if we have access to this stage_outputs. See deployment_fixtures.py line 201.